src/components/ArticleNavigation/ArticleNavigation.scss

```diff
@@ -0,0 +1,60 @@
+@use '~@gravity-ui/page-constructor/styles/variables.scss' as pcVariables;
+@use '../../variables.scss';
+
+$block: '.#{variables.$ns}article-navigation';
+
+#{$block} {
+    display: flex;
+    gap: var(--g-spacing-6);
+    @media (max-width: map-get(pcVariables.$gridBreakpoints, 'md') - 1) {
+        gap: var(--g-spacing-5);
+    }
+    &__content {
+        padding: var(--g-spacing-2) 0;
+        overflow: hidden;
+        width: 100%;
+        @media (max-width: map-get(pcVariables.$gridBreakpoints, 'md') - 1) {
+            padding: var(--g-spacing-3) 0;
+        }
+        &-title {
+            @media (max-width: map-get(pcVariables.$gridBreakpoints, 'md') - 1) {
+                display: none;
+            }
+        }
+    }
+    &__button {
+        cursor: pointer;
+        display: flex;
+        gap: var(--g-spacing-3);
+        width: 50%;
+        border-radius: 16px;
+        border: 1px solid var(--g-color-sfx-fade);
+        padding: var(--g-spacing-1);
+        justify-content: space-between;
+        &_reverse {
+            flex-direction: row-reverse;
+        }
+        &_right {
+            margin-left: auto;
+            padding-left: var(--g-spacing-5);
+        }
+        &_left {
```

Comment:
You don't need 3 classes to describe 2 button states
Here's what I've got:
Keep everything in 'left' within the button styles.
In '&_reverse', keep the override for flex-direction and padding.

It seems like margin is not needed at all. Manage buttons through flex rules in the container